packages/ceramic-common/src/utils/doctype-utils.ts

```diff
@@ -127,7 +127,7 @@ export class DoctypeUtils {
      */
     static makeReadOnly<T extends Doctype>(doctype: T): T {
         doctype.change = (): Promise<void> => {
-            throw new Error('The version of the document is readonly. Checkout the latest HEAD in order to update.')
+            throw new Error('The version of the document is readonly. Checkout the latest TIP in order to update.')
```

Comment:
I wasn't totally sure if it still made sense to keep 'TIP' capitalized in the places where 'HEAD' was, so let me know if you think 'Tip' would be better here

Reply:
I would change it to something like this:
```suggestion
            throw new Error('The version of the document is readonly. Load the document without version to make updates.')
```

Reply:
@oed yep, agree 👍

Reply:
Is the issue that the version specified is readonly, or that *any* specific version is inherently readonly?  Would a better message be: "Historical document versions cannot be modified. Load the document without specifying a version to make updates."?